utils/polus-bfio-util/bfio/bfio.py

```diff
@@ -1,457 +1,210 @@
-                pixels to load. If None, loads the full range.
-                Defaults to None.
-            Y ([tuple,list], optional): 2-tuple indicating the (min,max) range of
-                pixels to load. If None, loads the full range.
-                Defaults to None.
-            Z ([tuple,list], optional): 2-tuple indicating the (min,max) range of
-                pixels to load. If None, loads the full range.
-                Defaults to None.
-            C ([tuple,list], optional): tuple or list of values indicating channel
-                indices to load. If None, loads the full range.
-                Defaults to None.
-            T ([tuple,list], optional): tuple or list of values indicating timepoints
-                to load. If None, loads the full range.
-                Defaults to None.
+            X: The (min,max) range of pixels to load along the x-axis (columns).
+                If None, loads the full range. *Defaults to None.*
+            Y: The (min,max) range of pixels to load along the y-axis (rows). If
+                None, loads the full range. *Defaults to None.*
+            Z: The (min,max) range of pixels to load along the z-axis (depth).
+                If None, loads the full range. *Defaults to None.*
+            C: Values indicating channel indices to load. If None, loads the
+                full range. *Defaults to None.*
+            T: Values indicating timepoints to load. If None, loads the full
+                range. *Defaults to None.*
 
         Returns:
-            numpy.ndarray: A 5-dimensional numpy array.
-        
-        Example:
-            # Import javabridge and start the vm
-            import javabridge
-            javabridge.start_vm(class_path=bioformats.JARS)
-
-            # Path to bioformats supported image
-            image_path = 'path/to/image'
-
-            # Create the BioReader object
-            bf = BioReader(image_path)
-
-            # Load the full image
-            image = bf.read_image()
-
-            # Only load the first 256x256 pixels, will still load all Z,C,T dimensions
-            image = bf.read_image(X=(0,256),Y=(0,256))
-
-            # Only load the second channel
-            image = bf.read_image(C=[1])
-
-            # Done executing program, so kill the vm. If the program needs to be run
-            # again, a new interpreter will need to be spawned to start the vm.
-            javabridge.kill_vm()
+            A 5-dimensional numpy array.
         """
-
         # Validate inputs
         X = self._val_xyz(X, 'X')
         Y = self._val_xyz(Y, 'Y')
         Z = self._val_xyz(Z, 'Z')
         C = self._val_ct(C, 'C')
         T = self._val_ct(T, 'T')
-
-        # Set options for file loading based on metadata
-        # open in parts if more than max_bytes
-        open_in_parts = (X[1]-X[0])*(Y[1]-Y[0]) > self._pix['chunk']
+        
+        # Define tile bounds
+        X_tile_start = (X[0]//self._TILE_SIZE) * self._TILE_SIZE
```

Comment:
Unless there's something I'm missing, you have inconsistent use of hard-coded `1024` and `self._TILE_SIZE`